modules/mockserver/src/main/java/org/testcontainers/containers/MockServerContainer.java

```diff
@@ -36,7 +36,7 @@ public MockServerContainer(DockerImageName dockerImageName) {
         super(dockerImageName);
         dockerImageName.assertCompatibleWith(DEFAULT_IMAGE_NAME, DockerImageName.parse("mockserver/mockserver"));
 
-        waitingFor(Wait.forHttp("/mockserver/status").withMethod("PUT").forStatusCode(200));
+        waitingFor(Wait.forLogMessage(".*started on port: " + PORT + ".*", 1));
```

Comment:
this actually broke our podman rootless execution waiting for mockserver. We went back to the http status check. I wonder why using the default listening for the first exposed is not good enough?
```
DEBUG com.github.dockerjava.zerodep.shaded.org.apache.hc.client5.http.wire - http-outgoing-0 << "{"cause":"using --follow with the journald --log-driver but without the journald --events-backend (file) is not supported","message":"failed to obtain logs for Container '33ea4476017d4c49d84f2be6ec2eb7d0eae37ca064bdbab3c8771951f132ab2b': using --follow with the journald --log-driver but without the journald --events-backend (file) is not supported","response":500}[\n]"
```